APIs/openEO/federation/openeo_federation.md

```diff
@@ -0,0 +1,55 @@
+# Copernicus dataspace Ecosystem openEO federation
+
+The openEO standard offers a unique capability to provide access to datasets and processing capacity beyond the offering of CDSE.
+This gives you a single endpoint and interface for your workflows, avoiding the need to register with multiple platforms and
+study their APIs and offerings. This is referred to as the openEO federation, and is offered through a this endpoint:
+
+https://openeofed.dataspace.copernicus.eu
+
+This capability is currently in a phase where we encourage early adopters and projects with specific needs to try it out.
+It has already been extensively tested by users of other projects, but the configuration in the CDSE is new and thus we 
+offer it as a 'beta' service at first, allowing us to still make changes to the configuration based on user feedback.
+
+When using the federated endpoint, you will be asked to give consent for basic user information to be shared with the other providers
+that make up the federation, as well as to accept the terms and conditions. These external providers all comply with the 
+regulations of the federation, but are not necessarily part of the CDSE service. You can withdraw your consent at any time, via
```

Comment:
yes, this is a crucial but difficult part I believe. Need to find some better wording that is understandable and legally sound. Maybe just need to expand on it rather than trying to fit it into a single sentence.

Reply:
I tried to further improve. 
I'm wondering if we should mandate that all providers should come from EU countries, thus making sure that EU law applies.

Reply:
Good question, maybe we can await the feedback from ESA on this? As providers are requested to create a ticket, we can still evaluate this on a case-by-case basis.